extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -46,7 +47,11 @@ public static function getLangForPageShare( $browserLang, $useLang ) {
 		}
 	}
 
-	public static function isValidShareService( $service, $lang ) {
+	public static function isValidShareService( $service, $lang, $isTouchScreen ) {
+		//Don't display LINE social network (because it's a mobile app) on desktops
+		if ( $isTouchScreen === 'false' && ( $service['name'] === self::LINE_NAME ) ) {
```

Comment:
yes, it comes as a param - string. Thus evaluating both 'true' and 'false' strings as a bool values always gives true... so `!$isTouchScreen` was always evaluating to false.